File: solutions/clojure/sublist/1/src/sublist.clj
(ns sublist)

(defn classify [list1 list2] ;; <- arglist goes here
  (cond (= (compare list1 list2) 0) :equal
        (some #{list1} (partition (count list1) 1 list2)) :sublist
        (some #{list2} (partition (count list2) 1 list1)) :superlist
        :else :unequal
  )
)

File: solutions/javascript/sublist/1/sublist.js
//
// This is only a SKELETON file for the 'Sublist' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export class List {
  constructor(list = []) {
    this.list = list
  }

  compare(other) {
    let sup = this.list.toString().includes(other.list.toString())
    let sub = other.list.toString().includes(this.list.toString())
    switch (true) {
      case sub && sup: return 'EQUAL'
      case sub: return 'SUBLIST'
      case sup: return 'SUPERLIST'
      default: return 'UNEQUAL'
    }
  }
}
